staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/well_known_labels.go

```diff
@@ -34,12 +34,12 @@ const (
 	// fluentd in not running as a manifest pod with LabelFluentdDsReady.
 	LabelFluentdDsReady = "alpha.kubernetes.io/fluentd-ds-ready"
 
-	// When the --use-taint-based-evictions flag is enabled,
+	// When the --feature-gates=TaintBasedEvictions=true flag is enabled,
```

Comment:
Updated to `When feature-gate for TaintBasedEvictions=true flag is enabled,`, hope it is OK. Thanks @gmarek